### PR TITLE
Bind to adaptive.$ instead of Mobify.$

### PR DESCRIPTION
**Pull request by jnurse**
Use Adaptive.$ over Mobify.$ if it is defined

Status: **Opened for visibility**
Reviewers: **@scalvert @jansepar @donnielrt @kpeatt**

## Changes
- binds $ to Adaptive.$ if it's defined, otherwise falls back to Mobify.$ as before

### CODE REVIEW

File: src/magnifik.js [hunk 1]
```diff
@@ -1,7 +1,13 @@
 (function() {
 
-var Mobify = window.Mobify = window.Mobify || {}
-  , $ = Mobify.$ = Mobify.$ || window.$ || window.Zepto || window.jQuery;
+var Adaptive = window.Adaptive;
+var Mobify = window.Mobify = window.Mobify || {};
+
+if (Adaptive.$ === undefined) {
```
Comment (tedtate) [inline src/magnifik.js]: I think this is going to fail for projects where `window.Adaptive` does not exist.

**Comment by jnurse** [comment]
@tedtate You were right, it should work now.

**Comment by jansepar** [comment]
Apparently this is the branch everyone is using these days for magnifik :). :+1: !

**Comment by donnielrt** [comment]
@jansepar are you doing a new release too?

**Comment by jansepar** [comment]
@donnielrt Sonja has a PR coming down the pipe, figured I'll merge that first then do a release!

**Comment by donnielrt** [comment]
@jansepar cool, sounds good :) Thanks!

**Pull request merged**